neo/rawio/blackrockrawio.py

```diff
@@ -1870,6 +1888,17 @@ def __get_nonneural_evtypes_variant_b(self, data):
 
         return event_types
 
+    def __get_comment_evtypes_variant_a(self, data):
+        return {
+            'comments': {
+                'name': 'comments',
+                'field': 'comment',
+                'mask': np.asarray([len(_) > 0 for _ in data['comment']], dtype=bool)
```

Comment:
is there a reason to use this mask instead of directly comparing to the packet_id, which should be 65535 for comments?